x-pack/platform/plugins/shared/fleet/common/services/agentless_policy_helper.ts

```diff
@@ -92,7 +92,7 @@ export const isOnlyAgentlessIntegration = (
 export const isOnlyAgentlessPolicyTemplate = (policyTemplate: RegistryPolicyTemplate) => {
   return Boolean(
     policyTemplate.deployment_modes &&
-      policyTemplate.deployment_modes.agentless?.enabled === true &&
+      policyTemplate.deployment_modes.agentless.enabled === true &&
```

Comment:
Maybe we should keep this fix?
```suggestion
      policyTemplate.deployment_modes?.agentless.enabled === true &&
```